### PR TITLE
Fix typo in video URLs for the Everything Exhibition

### PR DESCRIPTION
**Pull request by blinry**

### CODE REVIEW

**Comment by blinry** [comment]
On second thought, it seems that <https://demoweek.prototypefund.de/assets/images/project_images/the-everything-exhibition/generate.mp4> with an "i" also doesn't work. Can you see what's going on here, @AnXh3L0?

**Comment by AnXh3L0** [comment]
Hi @blinry , the issue with the videos is not the path name, but the videos themselves, they are encoded badly so they don't play as normal .mp4-s. The only way to fix it is to re-encode the videos you provided to us so we can insert them again.

**Comment by blinry** [comment]
Huh. And generate.mp4 specifically just seems to be missing in the source directory. Investigating!

**Comment by blinry** [comment]
We think adding generate.mp4, and fixing the links should solve the problem. When running the Jekyll server locally, I can't see any embedded videos (also not the FOSDEM talk), but with a proper web server, it seems to work.

tl;dr: We hope that if you merge this PR, we are happy! :D

**Review by AnXh3L0 — APPROVED** [review]
